bin/ch-convert

```diff
@@ -86,7 +86,7 @@ cv_dir_chimage () {
     dir_in_validate "$1"
     chimage_out_validate "$2"
     INFO 'importing ...'
-    ch-image import "$1" "$2"  # FIXME: no progress meter
+    quiet_process ch-image import "$1" "$2"  # FIXME: no progress meter
```

Comment:
Since `ch-image` understands `-q` and `-v`, what about passing the arguments through?